spec/defines/requirements_spec.rb

```diff
@@ -33,7 +35,8 @@
           let :params do
             {
               owner: 'bob',
-              group: 'bob'
+              group: 'bob',
+              'cwd': '/foo'
```

Comment:
Isn't this valid too?
```suggestion
              cwd: '/foo'
```